Configuration/ProcessModifiers/python/PixelCPEGenericForBricked_cff.py

```diff
@@ -0,0 +1,4 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
I am not entirely sure I understand the case for this modifier.
Wouldn't it be possible to attach the changes for the generic algorithm directly to the other one `phase2_brickedPixels` ?

Reply:
Hi! 
We are having a chat with Ernesto and Matti at 17:30 h CERN time today, would you like to join so that we can discuss this issue ?
Franz

Reply:
The idea was to have the modifier phase2_brickedPixels devoted to the changes to the clusterizer and to the CPE generic algo needed for the bricked topology and a different modifier PixeCPEGenericForBricked to turn off template reco as we do in case of T22/2026D78 geometry.

e.g, PixelCPEGenericForBricked is (supposed to be) effective only in track reco while  phase2_brickedPixels is (supposed to be) effective already in local reco

Reply:
but at the moment you cannot even envisage a bricked pixels wf that doesn't run generic, isn't it?
And in any case if that limitation would be lifted in the future, you would just need to edit here: 

https://github.com/cms-sw/cmssw/blob/c25822b5611c2ed07de64dae4c768983b5b8acd2/RecoTracker/TransientTrackingRecHit/python/TTRHBuilderWithTemplate_cfi.py#L22-L23

and remove the modification.
I think for the sake of economy if would be better to bind everything to the same modifier.

Reply:
This was corrected in the last commit by doing what you suggested @mmusich